tests/e2e/e2e_test.go

```diff
@@ -375,9 +376,10 @@ var _ = Describe("odoe2e", func() {
 			runCmdShouldPass("odo login --username developer --password developer")
 			runCmdShouldPass("odo create php testcmp --app e2e-xyzk --git " + testPHPGitURL)
 			runCmdShouldPass("odo config set Ports 8080/TCP")
+			runCmdShouldPass("odo url create --port 8080")
```

Comment:
should verify url working? by curling it or something

Reply:
fixed